app/services/regards_sender.rb

```diff
@@ -1,29 +1,32 @@
 class RegardsSender
+  include Service
+
   def initialize(current_user, target_user_id)
     @current_user = current_user
     @target_user_id = target_user_id
   end
 
   def execute
-    return unless users_valid?
     parse_regards_params
+    return false unless users_valid?
     send_email
+    true
   end
 
   private
 
   attr_accessor :sender_user, :target_user, :current_user, :target_user_id
 
   def users_valid?
-    !current_user.nil? && !target_user_id.nil?
+    current_user && target_user
   end
 
   def parse_regards_params
     self.sender_user = current_user
-    self.target_user = User.find(target_user_id)
+    self.target_user = User.find_by(id: target_user_id)
```

Comment:
Yes, but `find` returns an exception, when record could not be found, while `find_by` returns nil, which makes method `execute` return false. This makes it possible to create proper flash message in case when target user does not exist. Also, makes it easier to test such scenario.